kubejs/startup_scripts/startup.js

```diff
@@ -693,7 +694,19 @@ onEvent("item.registry", (event) => {
         });
     event.create("createastral:pure_biomatter");
     event.create("createastral:lime").displayName("Lime Dust");
-    event.create("createastral:coin", "create:sequenced_assembly").displayName("coin");
+
+    // Transitional Items
```

Comment:
These don't have textures yet, these seem to come with https://github.com/Laskyyy/Create-Astral/pull/317.

This means that that PR is a blocker for this one.

Reply:
I will merge #317 into this branch. This will require merging #317 before merging this.